### PR TITLE
chore: update version to v0.7.0 and update the CRDs

### PR DESCRIPTION
**Pull request by iam-veeramalla**
**What type of PR is this?**
> /kind chore

**What does this PR do / why we need it**:
- Upgrade the version to `v0.7.0`
- Upgrade the CRDs 

**Have you updated the necessary documentation?**
NA

**How to test changes / Special notes to the reviewer**:
NA

### CODE REVIEW

**Comment by ishitasequeira** [comment]
@iam-veeramalla it seems the `make bundle` failed

**Review by ishitasequeira — APPROVED** [review]
LGTM!!

File: bundle/manifests/argocd-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -1189,4 +1189,4 @@ spec:
   provider:
     name: Argo CD Community
   replaces: argocd-operator.v0.5.0
```
Comment (jaideepr97) [inline bundle/manifests/argocd-operator.clusterserviceversion.yaml:1191]: @iam-veeramalla this needs to replace 0.6.0

**Comment by jaideepr97** [comment]
@iam-veeramalla shouldn't we update https://github.com/argoproj-labs/argocd-operator/blob/master/config/manifests/bases/argocd-operator.clusterserviceversion.yaml#L769 instead of updating the CSV directly in the bundle folder?

**Review by jaideepr97 — APPROVED** [review]
/lgtm 
Thanks @iam-veeramalla

**Pull request merged**